test/Util.js

```diff
@@ -79,10 +79,13 @@ exports.fillMap = function (map, size, keyPrefix, valuePrefix) {
     return map.putAll(entries);
 };
 
-exports.markEnterprise = function (_this) {
-    if (process.env.SERVER_TYPE === 'oss' || process.env.HZ_TYPE === 'oss') {
+exports.markCommunity = function (_this) {
+    if (process.env.HAZELCAST_ENTERPRISE_KEY) {
```

Comment:
In the current compatibility test suite, `HAZELCAST_ENTERPRISE_KEY` is always there, even when we run the OSS tests. I am not sure whether or not we are going to use the same test structure, but it might be good to be defensive here and check for `HZ_TYPE` also

Reply:
Makes sense. Reverted the change and applied the same logic to `markCommunity` in 0aa4108e004854f2f0a11af8281bb67d52a45ed5